openstack/compute/v2/extensions/attachinterfaces/requests.go

```diff
@@ -0,0 +1,13 @@
+package attachinterfaces
+
+import (
+	"github.com/gophercloud/gophercloud"
+	"github.com/gophercloud/gophercloud/pagination"
+)
+
+// ListInterfaces makes a request against the nova API to list the servers interfaces.
+func ListInterfaces(client *gophercloud.ServiceClient, id string) pagination.Pager {
```

Comment:
The attach-interfaces extension only lists one type of resource (interfaces), so this can be renamed to just `List`.

In addition, the `id` is the id of the server, so let's rename `id` to `serverID` so it's not mistaken as an "interface id".